neo4j-examples/paradise-papers-django
# Docs: Part 3

Closes [Ticket](https://trello.com)


Create draft for part 3 of the tutorial



## Testing

Steps to manually verify the change:
Setup: cd docs/
Run: make html
Verify: You can check the output of the above command in the _build/ directory
